<?xml version="1.0"?>
<launch>

	<node pkg="pcl_filters" type="downsampling_filter" name="downsampling_filter" output="screen" required="true"/>
  
</launch>
